provisioners/src/main/java/org/jboss/intersmash/provision/openshift/InfinispanOperatorProvisioner.java

```diff
@@ -73,31 +73,28 @@ public void deploy() {
 		ffCheck = FailFastUtils.getFailFastCheck(EventHelper.timeOfLastEventBMOrTestNamespaceOrEpoch(),
 				getApplication().getName());
 		subscribe();
-
-		// create custom resources
-		int replicas = getApplication().getInfinispan().getSpec().getReplicas();
+		// create Infinispan CR
```

Comment:
@fabiobrz out of curiosity, are these changes strictly related to the version upgrade or is a general improvement?

Reply:
The main goal is to bump the version but I pushed some minor fixes/cleanup as well.